docs/se/guides/tracing.adoc

```diff
@@ -232,12 +250,12 @@ image::guides/12_tracing_span_detail.png[Span Details]
 
 === Tracing Across Services
 
-Helidon automatically traces across services, providing that the services use the same tracer, for example, the same instance of Jaeger.
-This means a single trace can include spans from multiple services and hosts.  OpenTracing uses a `SpanContext` to
+Helidon automatically traces across services if the services use the same tracer, for example, the same instance of Jaeger.
+This means a single trace can include spans from multiple services and hosts.  Helidon uses a `SpanContext` to
 propagate tracing information across process boundaries.  When you make client API calls, Helidon will
-internally call OpenTracing APIs to propagate the `SpanContext`. There is nothing you need to do in your application to make this work.
+internally call OpenTelemetry APIs or OpenTracing APIs to propagate the Span Context. There is nothing you need to do in your application to make this work.
```

Comment:
If you are going to make any other changes, please change Span Context to either 

span context

or

`SpanContext` (with back ticks before and after)

Reply:
fixed